loco-new/base_template/src/controllers/auth.rs

```diff
@@ -26,6 +26,11 @@ pub struct ResetParams {
     pub password: String,
 }
 
+#[derive(Debug, Deserialize, Serialize)]
```

Comment:
somehow, only if easy i would add a way to validate an allow list for the emails.
for example, accept only emails from "work domains + gmail". which can avoid people using this to spam with fake emails.
then let the user define the function to validate the requested email (we can leave it open to their judgement)